app/models/source.js

```diff
@@ -8,47 +8,45 @@ var nameMethods = require('./methods/name');
 // a way to interporlate strings to get a final URL
 var templateCompiler = require('es6-template-strings');
 var SourceContentType = require('app/models/sourceContentType');
-var debug= require('debug');
 
 var methods = Object.assign({
-    itemDataObjectsFromPagePath: function(contentType) {
-        return templateCompiler(this.itemDataObjectsFromPagePathTemplate, {
-            contentTypePluralCamelName: contentType ? contentType.pluralCamelName() : undefined,
-            contentTypePluralLowercaseName: contentType ? contentType.pluralLowercaseName() : undefined
-        });
-    },
+  itemDataObjectsFromPagePath: function (contentType) {
+    return templateCompiler(this.itemDataObjectsFromPagePathTemplate, {
+      contentTypePluralCamelName: contentType ? contentType.pluralCamelName() : undefined,
+      contentTypePluralLowercaseName: contentType ? contentType.pluralLowercaseName() : undefined
+    });
+  },
 
-    totalItemsAvailableFromPagePath: function(contentType) {
-        return templateCompiler(this.totalItemsAvailableFromPagePathTemplate, {
-            contentTypePluralCamelName: contentType ? contentType.pluralCamelName() : undefined,
-            contentTypePluralLowercaseName: contentType ? contentType.pluralLowercaseName() : undefined
-        });
-    },
+  totalItemsAvailableFromPagePath: function (contentType) {
```

Comment:
This appears to be the rule we need to use for declarations: https://eslint.org/docs/rules/space-before-function-paren